client/src/components/search/searchBar/SearchHits.js

```diff
@@ -3,48 +3,75 @@ import { connectStateResults, connectHits } from 'react-instantsearch-dom';
 import isEmpty from 'lodash/isEmpty';
 import Suggestion from './SearchSuggestion';
 
-const CustomHits = connectHits(({ hits, currentRefinement, handleSubmit }) => {
-  const shortenedHits = hits.filter((hit, i) => i < 8);
-  const defaultHit = [
-    {
-      objectID: `default-hit-${currentRefinement}`,
-      query: currentRefinement,
-      _highlightResult: {
-        query: {
-          value: `
+const CustomHits = connectHits(
+  ({
+    hits,
+    currentRefinement,
+    handleSubmit,
+    handleHits,
+    handleMouseEnter,
+    handleMouseLeave
+  }) => {
+    const footer = [
+      {
+        objectID: `default-hit-${currentRefinement}`,
+        query: currentRefinement,
+        _highlightResult: {
+          query: {
+            value: `
             See all results for
             <ais-highlight-0000000000>
             ${currentRefinement}
             </ais-highlight-0000000000>
           `
+          }
         }
       }
-    }
-  ];
-  return (
-    <div className='ais-Hits'>
-      <ul className='ais-Hits-list'>
-        {shortenedHits.concat(defaultHit).map(hit => (
-          <li
-            className='ais-Hits-item'
-            data-fccobjectid={hit.objectID}
-            key={hit.objectID}
-          >
-            <Suggestion handleSubmit={handleSubmit} hit={hit} />
-          </li>
-        ))}
-      </ul>
-    </div>
-  );
-});
+    ];
+    const allHits = hits.filter((_, i) => i < 8).concat(footer);
```

Comment:
```suggestion
    const allHits = hits.slice(0, 8).concat(footer);
```
I think we might as well use slice, right?  Just a bit neater.